app/uk/gov/hmrc/gform/sharedmodel/formtemplate/Section.scala

```diff
@@ -96,6 +96,7 @@ object Section {
     title: SmartString,
     description: SmartString,
     shortName: SmartString,
+    summaryName: Option[SmartString],
```

Comment:
Shouldn't this be mandatory?

Reply:
I have left it as Optional, so it works for existing form templates already saved in database.

Reply:
We don't need to keep backward functionality. That is why we have "Reload All" functionality in Admin tool.

Reply:
Cool, will make the change then